package.json

```diff
@@ -20,7 +20,7 @@
     "lib"
   ],
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0"
```

Comment:
dropping old node. shouldn't this be semver-major ?

Reply:
because we don’t have a major version, then by seven versions this is like major

Reply:
this make sense 👍